### ISSUE TITLE
what do the used colors mean?

### ISSUE THREAD

**Issue by staabm** [TomasVotruba/friendsofphp.org#170]
there is no explanation what the colored used in the website indicate

![image](https://user-images.githubusercontent.com/120441/76527154-079b7f00-646f-11ea-8639-e276000fbeca.png)

**Comment by TomasVotruba** [comment]
It's like traffix lights:

- next week - green
- 2 weeks - orange
- futher - red

How could we make it more obvious?

**Comment by staabm** [comment]
maybe just add some kind of "legend" which desribes the colors?

**Comment by TomasVotruba** [comment]
Could you PR it?

**Comment by staabm** [comment]
I though about somethink like

![grafik](https://user-images.githubusercontent.com/120441/77165932-92a1f800-6ab3-11ea-945b-2b1ae47f6d1b.png)


you just need to add
```
<tr>
        <th colspan="3" style="text-align: center;">next week</th>
    </tr>
```

...

```
<tr>
        <th colspan="3" style="text-align: center;">in 2 weeks</th>
    </tr>
```

or similar.

I am not able to express this in TWIG so

**Comment by TomasVotruba** [comment]
I don't think it would be possible, because we'd have to filter number of lines of specific colors. That heavy on JavaScript computing and I have troubles writing it in a good way.

What about just legend above the table?

**Comment by jpalala** [comment]
How about putting in human readable dates (for example -  it would automatically say: instead of the date, the phrase "in X days/Xweeks/") ? [MomentJS](https://momentjs.com/) is the most popular library that could be used for this, however there are probably lighter versions for human readable date. 

The UI would have to be revised though giving importance to the title and then the human readable date.

**Comment by TomasVotruba** [comment]
Hi, picture is worth thousand words. Could you make a prototype PR?

**Comment by jpalala** [comment]
I'll try to file a PR if I have time this weekend. It looks like I'll have to add [Carbon](https://carbon.nesbot.com/docs/#api-humandiff) package instead of the MomentJS libray.

The code would look something like this:

```php
<tr class="meetup odd" data-longitude="138.59996032715" data-latitude="-34.928619384766" data-datetime="2021-04-15 18:00" role="row">
    <td class="w-75">
                    <span class="text-nowrap text-success">Laravel.PH<br>
                    </span>

        <br>

        <small class="text-secondary"><?php echo Carbon::parse(date('Y-m-d'))->diffForHumans(date(Y-m-d, /* timestamp of the meetup*/));  ?> <!-- (5 days from now) --> at <?php echo date('H:i'); ?></small>
    </td>
    <td class="w-25">
        <a href="#" onclick="relocate(-34.928619384766, 138.59996032715)">Manila
</a>

        <br>

        <small class="text-secondary">Philippines</small>
    </td>
    
</tr>
```

Here's what it could look like (see first line - wasn't able to finish editing the second line's meetup 😄 ) :

<img width="375" alt="Example-FriendsOfPHP-Using-Moment" src="https://user-images.githubusercontent.com/622659/114157841-a5090100-9956-11eb-8fb3-d605e4510477.png">

**Comment by TomasVotruba** [comment]
Looks good :) the absolute date should be there too, so it's easier to add it calendar

**Comment by jpalala** [comment]
I did my best, but the  tests/checks got me! See PR #182

**Comment by TomasVotruba** [comment]
Closing as answered
